ltr/concourse/pipeline.yaml

```diff
@@ -109,6 +109,13 @@ jobs:
       trigger: true
     - get: search-api-git
     - task: fetch
+      ensure:
+        params:
+          GOVUK_ENVIRONMENT: integration
+          ROLE_ARN: ((integration-role-arn))
+        run:
+          path: bash
+          args: ["search-api-git/ltr/concourse/after-fetch.sh"]
```

Comment:
Looks good. I think you need to test the `params` and `run` bits under a `config:` (or put it all in a `file:` if you don't want to duplicate stuff). You'll also need to define the `platform` and `image_resource` you want the task to use, and the step should be `task` here.

This should work:

```yaml
      ensure:
        task: teardown
        config:
          <<: *task-config
          inputs:
          - name: search-api-git
          params:
            GOVUK_ENVIRONMENT: integration
            ROLE_ARN: ((integration-role-arn))
          run:
            path: bash
            args: ["search-api-git/ltr/concourse/after-fetch.sh"]
```

You can use `fly validate-pipeline -c ltr/concourse/pipeline.yaml` to check it - but I should really put that in a GitHub Action!